package.json

```diff
@@ -77,7 +77,7 @@
   },
   "optionalPeerDependencies": {
     "abavalidator": ">=2 <3",
-    "bignumber.js": ">=2 <3",
+    "bignumber.js": ">=7",
```

Comment:
Add `<9`, to avoid possible breaking changes to this code?

Reply:
👍